### PR TITLE
Cleanup - Add braintree specific config for local development

### PR DESCRIPTION
**Pull request by jferm**
Adds a development environment configuration to streamline Braintree-internal setup and development for local integrations.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -112,14 +119,22 @@
...
 def _oauth_gateway
   Braintree::Gateway.new({
     :client_id => ENV["CLIENT_ID"],
     :client_secret => ENV["CLIENT_SECRET"],
-    :environment => "sandbox",
+    :environment => BRAINTREE_ENVIRONMENT,
+  })
+end
+
+def _api_key_oauth_gateway
```
Comment (aweiksnar) [inline app.rb]: It looks like this method is unused?

File: .development.env [hunk 2]
```diff
@@ -0,0 +1,7 @@
+PORT='9393'
```
Comment (ThatHurleyGuy) [inline .development.env]: I would vote we just always run the app on this port. We can tweak the heroku settings but I can't think of any reason we really need to be switching between ports based on environments

File: .development.env [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+CLIENT_ID='client_id$development$integration_client_id'
+CLIENT_SECRET='client_secret$development$integration_client_secret'
+REDIRECT_URI="http://$BT_DEV_HOST:9393/callback"
```
Comment (ThatHurleyGuy) [inline .development.env]: I'd be all in favor of changing this in our populate_data script to get rid of the inconsistency of `/cb` and `/callback`

**Review by ThatHurleyGuy — APPROVED** [review]
⚡️

**Review by aweiksnar — APPROVED** [review]
🌚

**Pull request merged**